packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -41,15 +43,11 @@ class DropdownProps {
   position = prop<string>({ type: String, default: '' })
   hoverOverTimeout = prop<number>({ type: Number, default: 30 })
   hoverOutTimeout = prop<number>({ type: Number, default: 200 })
-  boundaryBody = prop<boolean>({ type: Boolean })
   modelValue = prop<boolean>({ type: Boolean, default: false })
   disabled = prop<boolean>({ type: Boolean })
-  // Makes no sense
-  // fixed = prop<boolean>({ type: Boolean })
+  target = prop<string>({ type: String, default: '#app' })
```

Comment:
What's this magical `#app`?
Maybe use `body` instead?

We likely want this component to work fine even if there is no `#app`.